cluster/stone-prd-rh01/tenants/insights-core-tenant/insights-puptoo-app-bonfire-tekton.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: appstudio.redhat.com/v1beta1
```

Comment:
I recommend using latest version of crd which is v1beta2.

Reply:
Updated